app/model/activity_program.go

```diff
@@ -0,0 +1,16 @@
+package model
+
+type ActivityProgram struct {
+	ID               int
```

Comment:
gormの仕様上、PKにはオプションを追加してください。

```suggestion
	ID               int `gorm:"primaryKey"`
```